20230726/s6-1690380087112/README.md

```diff
@@ -0,0 +1,61 @@
+# Scenario 6 - ALS Baseline ALS Baseline with Sims-only, HTTP-Keep-Alive enabled
```

Comment:
Repeat of ALS Baseline and request to update from Sims-only to Sims.

Reply:
Noted